scripts/servers.py

```diff
@@ -755,24 +755,28 @@ def managed_acceptance_tests_server(
         Exception. The suite_name is not in the list of the acceptance tests
             suite names.
     """
+    if suite_name not in common.ACCEPTANCE_TESTS_SUITE_NAMES:
+        raise Exception('Invalid suite name: %s' % suite_name)
+
     nodemodules_jasmine_bin_path = os.path.join(
         common.NODE_MODULES_PATH, '.bin', 'jasmine')
     puppeteer_acceptance_tests_dir_path = os.path.join(
         common.CURR_DIR, 'core', 'tests', 'puppeteer-acceptance-tests')
+    build_dir_path = os.path.join(
```

Comment:
Maybe let's call this puppeteer_build_dir_path, since we use the concept of "build" elsewhere in the codebase as well.

Reply:
Done